apps/web/app/(admin)/members/_schemas/Form.schema.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from "zod";
+import { type StaticImageData } from "next/image";
+import { ROLE_LABELS } from "@repo/types/src/membersType";
+import { MEMBER_FORM_MESSAGES } from "@repo/constants/messages";
+
+export const memberFormSchema = z.object({
```

Comment:
```suggestion
export const MembersSchema = z.object({
```
schema는 파스칼케이스가 더 알맞는거같고
```
export type MembersType = z.infer<typeof MembersSchema>
```
이런식으로 type 뽑아서 사용하는게 zod사용할때 더 범용성있는거같아